### PR TITLE
Switch to PSTT live, close #94

### LINKED ISSUE #94: Switch to PSTT live

### PR DESCRIPTION
**Pull request by bennuttall**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**

- Closes #94 

**State whether the PR is ready for review or whether it needs extra work**    

- Needs deploying and testing, but should be good to go

### CODE REVIEW

**Comment by bennuttall** [comment]
Deployed and tested - @allishultes can you review? This is now mergeable

File: functions/awsUploader/index.js [hunk 1]
```diff
@@ -10,34 +10,14 @@ exports.createHandler = async (snap, bucket, aws, context) => {
   const metadata = snap.data();
   const durationSeconds = Math.ceil(metadata.duration);
   const fileSize = metadata.size;
-  const params = {
```
Comment (allishultes) [inline functions/awsUploader/index.js:13]: What does it mean that we've lost duration and the expiration field? Do you know how those were being used before?

File: functions/awsUploader/index.js [hunk 2]
```diff
@@ -10,34 +10,14 @@ exports.createHandler = async (snap, bucket, aws, context) => {
   const metadata = snap.data();
   const durationSeconds = Math.ceil(metadata.duration);
```
Comment (allishultes) [inline functions/awsUploader/index.js:11]: This doesn't look like it's being used in this file anymore - which, if that's ok, means I'd probably not define here.

File: functions/utils/aws.js [hunk 3]
```diff
@@ -1,36 +1,62 @@
...
-      accessKeyId: aws.key,
-      secretAccessKey: aws.secret,
-    });
-    s3.getSignedUrl("putObject", params, (err, url) => {
-    if (err) {
-      console.log("[ERROR] Error getting signed url", err);
-      reject(err)
-    }
+const getSignedUrl = async (aws, fileName, durationSeconds) => {
+  const headers = {
```
Comment (allishultes) [inline functions/utils/aws.js:4]: Do you have the linter working in these files? Nit-picky but there are a mix of different quote styles used. Do you need doubles for headers?

Comment (bennuttall) [reply functions/utils/aws.js:4]: Not sure my linter is configured correctly, I'll fix the quotes

**Pull request merged**